src/main/java/seedu/jarvis/ui/MainWindow.java

```diff
@@ -244,6 +253,16 @@ private void setGreeting() {
         greeting.textProperty().bind(logic.getGreeting());
     }
 
+    /**
+     * Sets the details for a summary of the number of ungraded missions, ungraded quests, consultations, tasks and
+     * mastery checks.
+     */
+    private void setSummary() {
+        // check if user logged in.
```

Comment:
I feel that getting values from various lists does not need to be concerned with whether user is logged in or not.